mods/gen4/formats-data.js

```diff
@@ -1942,19 +1942,19 @@ let BattleFormatsData = {
 		tier: "OU",
 	},
 	rotomwash: {
-		randomBattleMoves: ["thunderbolt", "substitute", "painsplit", "shadowball", "hiddenpowerice", "willowisp", "trick", "hydropump"],
+		randomBattleMoves: ["thunderbolt", "substitute", "painsplit", "shadowball", "hiddenpowerice", "willowisp", "trick", "hydropump", "rest", "sleeptalk"],
 		tier: "OU",
 	},
 	rotomfrost: {
-		randomBattleMoves: ["thunderbolt", "hiddenpowerfighting", "shadowball", "trick", "blizzard", "substitute", "painsplit"],
+		randomBattleMoves: ["thunderbolt", "shadowball", "trick", "blizzard", "substitute", "painsplit", "willowisp", "rest", "sleeptalk"],
```

Comment:
See above.

Reply:
Same. Shadow Ball, Blizzard, and WoW all seem fine alongside RestTalk and Thunderbolt. That's one of the reasons I removed HP Fighting though. Rest/Talk/Tbolt/HPFight would be pretty bad.